src/utilcode/clrhost_nodependencies.cpp

```diff
@@ -17,6 +17,12 @@
 #include "contract.h"
 #include "tls.h"
 
+#if defined PLATFORM_UNIX
```

Comment:
This should rather be #ifdef __llvm__ . You are taking advantage of llvm-specific feature, not Unix-specific feature.